spring-cloud-common-security-config-web/src/main/java/org/springframework/cloud/common/security/OAuthSecurityConfiguration.java

```diff
@@ -106,272 +108,440 @@
 @ConditionalOnWebApplication(type = ConditionalOnWebApplication.Type.ANY)
 @EnableWebSecurity
 @Conditional(OnOAuth2SecurityEnabled.class)
+@Import({
+        OAuthSecurityConfiguration.OAuth2AccessTokenResponseClientConfig.class,
+        OAuthSecurityConfiguration.OAuth2AuthenticationFailureEventConfig.class,
+        OAuthSecurityConfiguration.OpaqueTokenIntrospectorConfig.class,
+        OAuthSecurityConfiguration.OidcUserServiceConfig.class,
+        OAuthSecurityConfiguration.PlainOauth2UserServiceConfig.class,
+        OAuthSecurityConfiguration.WebClientConfig.class,
+        OAuthSecurityConfiguration.AuthoritiesMapperConfig.class,
+        OAuthSecurityConfiguration.OAuth2TokenUtilsServiceConfig.class,
+        OAuthSecurityConfiguration.LogoutSuccessHandlerConfig.class,
+        OAuthSecurityConfiguration.ProviderManagerConfig.class,
+        OAuthSecurityConfiguration.AuthenticationProviderConfig.class
+})
 public class OAuthSecurityConfiguration extends WebSecurityConfigurerAdapter {
 
-	private static final org.slf4j.Logger logger = LoggerFactory.getLogger(OAuthSecurityConfiguration.class);
```

Comment:
Did you also do a file reformat in here? It is very hard to see what has actually changed in Github diff view.

I will assume the changes are all around wrapping the beans into configs and no real shift in logic. If that is not the case, please lmk the locations.

Reply:
The `.editorconfig` has tabs configured so a reformat my replace old spaces with tabs.

Reply:
What does the Spring Cloud build tools style suggest for this?

Reply:
Also, please answer 

> I will assume the changes are all around wrapping the beans into configs and no real shift in logic. If that is not the case, please lmk the locations.